drivers/iio/frequency/ad917x/ad917x_jesd_api.c

```diff
@@ -243,6 +243,52 @@ int ad917x_jesd_get_cfg_param(ad917x_handle_t *h,
 	return API_ERROR_OK;
 }
 
+int ad917x_jesd_oneshot_sync(ad917x_handle_t *h, u8 err_window)
+{
+	int err;
+	u8 mode;
+
+	if (h == NULL)
```

Comment:
```suggestion
	if (!h)
```

same for all instances